zerver/migrations/0327_alter_realm_message_content_delete_limit_seconds.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.4 on 2021-06-14 12:12
+
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def make_zero_invalid_for_message_delete_limit(
+    apps: StateApps, schema_editor: DatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    Realm.DEFAULT_MESSAGE_CONTENT_DELETE_LIMIT_SECONDS = 600
+
+    Realm.objects.filter(
+        allow_message_deleting=True, message_content_delete_limit_seconds=0
+    ).update(message_content_delete_limit_seconds=None)
+
+    Realm.objects.filter(
+        allow_message_deleting=False, message_content_delete_limit_seconds=0
+    ).update(
+        message_content_delete_limit_seconds=Realm.DEFAULT_MESSAGE_CONTENT_DELETE_LIMIT_SECONDS
+    )
```

Comment:
Have just made the value of `message_content_delete_limit_seconds` as its value eventually not matters as `allow_message_deleting=False`.